src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -0,0 +1,358 @@
+# Command-line arguments
+
+Here's the list of arguments you can pass to `rustdoc`:
```

Comment:
@rust-lang/cargo , is this something that's expected, or a bug? I'm kinda surprised, but I'm sure there's a good reason.